framework/mesh/mesh_vector.h

```diff
@@ -19,298 +20,157 @@ struct TensorRank2Dim3;
  */
 struct Vector3
 {
-  /// Element-0
-  double x;
-  /// Element-1
-  double y;
-  /// Element-2
-  double z;
-
-  /// Default constructor. Initialized as all zeros.
-  Vector3()
-  {
-    x = 0.0;
-    y = 0.0;
-    z = 0.0;
-  }
+  double x{0.0}; // X-component of the vector
+  double y{0.0}; // Y-component of the vector
+  double z{0.0}; // Z-component of the vector
```

Comment:
This removes the doxygen comments (doxygen ignores `//`). So if you want that to be documented, this needs to be `///` and on the line before the variable...

Reply:
done